### PR TITLE
Enable create tests and adapt it for the new primitive prototype behavior

### PR DESCRIPTION
**Pull request by blikblum**
It enables the create tests and adapt it to match new behavior

Had to change how created objects are compared because node `assert.deepStrictEqual` takes into consideration to prototype

For falsey and primitive as prototype, just check if is a object

Let me know if there's a better way to handle these tests

### CODE REVIEW

File: test/create.test.js [hunk 1]
```diff
@@ -55,23 +64,23 @@ describe('create', function() {
...
-
+  it('should accept a primitive `prototype`', function() {
     var actual = lodashStable.map(primitives, function(value, index) {
-      return lodashStable.isPlainObject(index ? create(value) : create());
+      return index ? create(value) : create();
     });
 
-    assert.deepStrictEqual(actual, expected);
+    actual.forEach(value => {
+      assert.ok(value && typeof value === 'object');
```
Comment (jdalton) [inline test/create.test.js]: You can use `lodashStable.isObject()` there.

File: test/create.test.js [hunk 2]
```diff
@@ -43,7 +46,13 @@ describe('create', function() {
     }
     Foo.prototype.b = 2;
 
-    assert.deepStrictEqual(create({}, new Foo), { 'a': 1, 'c': 3 });
+    var actual = create({}, new Foo);
+    var expected = { 'a': 1, 'c': 3 };
+
+    assert.deepStrictEqual(Object.keys(actual), Object.keys(expected));
+    Object.keys(expected).forEach(property => {
```
Comment (jdalton) [inline test/create.test.js]: I see in a couple places we do `Object.keys(expected)` would it make sense to store the result in a var when possible?

File: test/create.test.js [hunk 3]
```diff
@@ -33,7 +33,10 @@ describe('create', function() {
 
     assert.ok(actual instanceof Circle);
     assert.ok(actual instanceof Shape);
-    assert.deepStrictEqual(Circle.prototype, expected);
+    assert.deepStrictEqual(Object.keys(Circle.prototype), Object.keys(expected));
+    Object.keys(expected).forEach(property => {
```
Comment (jdalton) [inline test/create.test.js]: arrow functions should wrap the param in parens (style consistency nit)

Comment (blikblum) [reply test/create.test.js]: done

File: test/create.test.js (hunk 2 above)
Comment (blikblum) [reply test/create.test.js]: done

**Comment by jdalton** [comment]
Thanks @blikblum!

**Pull request merged**